### PR TITLE
B4 history and authentication

### PR DESCRIPTION
**Pull request by sumitk-singh**
added the history page 
added the auth feature

please review the code.

### CODE REVIEW

File: src/pages/Auth/SignUp/SignUp.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    width: 100%;
+  }
+  
+  .signup-inputs {
+    height: fit-content;
+    padding: 1rem;
+    font-size: small;
+  }
+  
+  .user-link {
```
Comment (aiman-ansari) [inline src/pages/Auth/SignUp/SignUp.css]: Suggestion: If this code is not necessary then remove it.

File: src/pages/Auth/SignUp/SignUp.jsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
+
+import './SignUp.css';
+import { useState } from "react";
+import { useEffect } from "react";
```
Comment (aiman-ansari) [inline src/pages/Auth/SignUp/SignUp.jsx]: You can import both of them into single line

File: src/pages/Auth/SignUp/SignUp.jsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                  }
+                />
+                        
+                <button className="create-account-btn" style={{cursor:"pointer"}} type="submit" >
+                  Create Account
+                </button>
+            
+
+            
+                <a style={{cursor:"pointer"}} onClick={() => navigate("/login")}>Already a user? Login</a>
```
Comment (aiman-ansari) [inline src/pages/Auth/SignUp/SignUp.jsx]: I think you forgot to remove a tag here.

File: src/pages/History/History.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+                <section className='section-b'>
+                    
+                    <h2>History <span className='float-right'><button>Clear Full History</button></span></h2>
+                    
+                    <section className='history-page-section-1'>
+                        
+                    {
+                            history.map((video) => {
+                                return <VideoRemoveCard id={video.id} thumbnail={video.image} title={video.title} views={video.views} xDaysAgo={video.date} video={video} type={ "REMOVE_FROM_HISTORY"}/>
```
Comment (aiman-ansari) [inline src/pages/History/History.jsx:28]: Suggestion: Here you can destructure video

Comment (aiman-ansari) [inline src/pages/History/History.jsx:28]: Suggestion: Here you can destructure video

**Review by aiman-ansari — APPROVED** [review]
Remove extra spacing use prettier for better code indentation.
Some suggestions are given do check.

File: src/pages/Auth/SignUp/SignUp.jsx (hunk 2 above)
Comment (sumitk-singh) [reply src/pages/Auth/SignUp/SignUp.jsx]: yeah, right

**Pull request merged**